src/components/carousel/_index.scss

```diff
@@ -0,0 +1,152 @@
+@use "../../scss";
+@use "~slick-carousel/slick/slick";
+
+.c-carousel {
+	@include scss.component-properties("carousel");
+
+	/* Arrows */
+	.slick-prev,
+	.slick-next {
+		cursor: pointer;
+		font-size: 0;
+		line-height: 0;
+		position: absolute;
+		top: 50%;
+		display: block;
+
+		&:hover {
+			color: transparent;
+			outline: none;
+			background: transparent;
+
+			&:before {
+				opacity: 1;
+			}
+		}
+
+		&:focus {
+			color: transparent;
+			outline: none;
+			background: transparent;
+
+			&:before {
+				opacity: 1;
+			}
+		}
+
+		@include scss.component-properties("carousel-buttons");
+	}
+
+	.slick-next {
+		@include scss.component-properties("carousel-button-next");
+
+		&:before {
+			font-family: "Arial", serif;
+			content: ">";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+
+			@include scss.component-properties("carousel-button-next-before");
+		}
+	}
+
+	.slick-prev {
+		@include scss.component-properties("carousel-button-previous");
+
+		&:before {
+			font-family: "Arial", serif;
+			content: "<";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+
+			@include scss.component-properties("carousel-button-previous-before");
+		}
+	}
+
+	.slick-prev.slick-disabled {
+		&:before {
+			opacity: 0.25;
+		}
+	}
+
+	.slick-next.slick-disabled {
+		&:before {
+			opacity: 0.25;
+		}
+	}
+
+	[dir="rtl"] {
+		.slick-prev {
+			@include scss.component-properties("carousel-button-previous");
+			left: auto;
+
+			&:before {
+				content: ">";
+			}
+		}
+
+		.slick-next {
+			right: auto;
+			@include scss.component-properties("carousel-button-next");
+
+			&:before {
+				content: "<";
+			}
+		}
+	}
+
+	/* Dots */
+	.slick-dotted.slick-slider {
+		@include scss.component-properties("carousel-slider-bottom");
+	}
+
+	.slick-dots {
+		list-style: none;
+		display: block;
+
+		@include scss.component-properties("carousel-slider-dots");
+
+		li {
+			cursor: pointer;
+
+			@include scss.component-properties("carousel-slider-dots-list");
+
+			button {
+				@include scss.component-properties("carousel-slider-dots-list-buttons");
+
+				&:hover {
+					outline: none;
+
+					&:before {
+						opacity: 1;
+					}
+				}
+
+				&:focus {
+					outline: none;
+
+					&:before {
+						opacity: 1;
+					}
+				}
+
+				&:before {
+					font-family: "Arial", serif;
+					content: "⋅";
+					-webkit-font-smoothing: antialiased;
+					-moz-osx-font-smoothing: grayscale;
+
+					@include scss.component-properties("carousel-slider-dots-list-buttons-before");
+				}
+			}
+		}
+
+		li.slick-active {
+			button {
+				&:before {
+					opacity: 0.75;
```

Comment:
Not allowing a theme override for active item opacity?
```
@include scss.component-with-states("carousel-slider-active-dot-buttons", [before])
```
at the end of the `li.slick-active button {}` block

Reply:
@nschubach Can you explain the `[before]` syntax ...especially how its in brackets